web/vtctld2/src/app/dashboard/shard.component.ts

```diff
@@ -72,20 +65,55 @@ export class ShardComponent implements OnInit, OnDestroy {
     });
   }
 
+  getActions() {
+    let result: MenuItem[] = [
+      {label: 'Status', items: [
+        {label: 'Validate', command: (event) => {this.openValidateShardDialog(); }},
+        {label: 'Validate Version', command: (event) => {this.openValidateVerShardDialog(); }},
```

Comment:
I need to just remove the first Validate, it's being replaced by the topo validator. I'll leave it as is now, until it's remove.

(also I don't understand the change you're suggesting, rename the first "Validate" when it's already named Validate?).